tf2_geometry_msgs/test/test_tf2_geometry_msgs.cpp

```diff
@@ -43,16 +43,16 @@ static const double EPS = 1e-3;
 
 TEST(TfGeometry, Frame)
 {
-  geometry_msgs::PoseStamped v1;
+  geometry_msgs::msg::PoseStamped v1;
   v1.pose.position.x = 1;
   v1.pose.position.y = 2;
   v1.pose.position.z = 3;
   v1.pose.orientation.x = 1;
-  v1.header.stamp = builtin_interfaces::msg::Time(2);
+  v1.header.stamp = tf2::TimePoint(2);
```

Comment:
you're right, it was an over-eager find-and-replace. I've changed it back to how it was in https://github.com/ros2/geometry2/pull/7/commits/3bcf95dcb2731e0fcfdd3b2e14930ad903085d23 but even that constructor isn't available I think. The tests haven't been ported yet but you're right that it's best to not make things worse!